FlorenceBML/BMLor321/BMLor321.xml

```diff
@@ -34,8 +34,133 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         <altIdentifier>
                             <idno>Marrassini ms. 21</idno>
                         </altIdentifier>
+                        <altIdentifier>
+                            <idno>Assemani CCCCV</idno>
+                        </altIdentifier>
                     </msIdentifier>
+                    <msContents>
+                        <summary/>
+                        <msItem xml:id="ms_i1">
+                            <locus from="1r" to="68v"/>
+                            <title>Gǝʿǝz-Latin Lexicon</title>
```

Comment:
I think you need to create an ID, of the type Studies

Reply:
OK. I expect this will be very similar to a work IK.

Reply:
Yes it is the same as work but gets a different prefix (STU not LIT).